freqtrade/rpc/telegram.py

```diff
@@ -354,19 +355,39 @@ def _status_table(self, update: Update, context: CallbackContext) -> None:
         :return: None
         """
         try:
+            fiat_currency = self._config.get('fiat_display_currency', '')
             statlist, head = self._rpc._rpc_status_table(
-                self._config['stake_currency'], self._config.get('fiat_display_currency', ''))
+                self._config['stake_currency'], fiat_currency)
+
+            show_total = fiat_currency != ''
+            total_sum = 0.0
+            if show_total:
+                r = re.compile(".*\\((-?\\d*\\.\\d*)\\)")
+                for trade in statlist:
+                    m = r.match(trade[-1])
+                    if m is not None:
+                        total_sum += float(m[1])
```

Comment:
as a VERY quick feedback:
no matter what you're trying to do - this is not the right approach as it'll be very error-prone - and you're trying to parse this out from a string that was created less than a second ago (or 5 lines).

Maybe consider moving the calculation of the sum to `rpc.py` - and returning it as additional field (`_rpc_status_table` is only used by telegram afaik - so the impact should be pretty low).

Reply:
![image](https://user-images.githubusercontent.com/147348/118492743-ff9d3600-b728-11eb-9b38-a504745d9bdf.png)

Reply:
This is how it's going to look if number of active trades is more than one

Reply:
Overall - i like.

is there a particular reason you chose not to show the relative profits - but only the Fiat one?
(note: this is more a question about your thoughts / considerations, not a request to add it).

Reply:
Agree, it wasn't possible when I parsed the output of _rpc_status_table command to get total profit but now, when logic has moved to the RPC method, it could be implemented